02-express-tutorial/routes/people.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const router = express.Router();
+const { addPerson, getPeople, getPerson, updatePeople, deletePerson } = require("../controllers/people.js");
+
+router.get('/', getPeople);
+router.get('/:id', getPerson);
+router.post('/', addPerson);
+router.put('/:id', updatePeople);
+router.delete('/:id', deletePerson);
```

Comment:
Nitpick (doesn't require changes):
When naming endpoints, I like to prefix them with an action, so it's easier to read/understand what the route expects to happen. 

For example:
```js
// instead of...
// http://localhost:3000/api/v1/people/6
router.delete('/:id', deletePerson);

// it becomes...
// http://localhost:3000/api/v1/people/delete/6
router.delete('/delete/:id', deletePerson);
```